webcompat/static/css/development/components/icon-font.css

```diff
@@ -13,8 +13,16 @@
 .wc-Icon {
   display: inline-block;
   vertical-align: middle;
-  font: normal normal 400 normal 1em/1 webcompat;
+
+  /* to prevent a webcompat issue, we do not use the short for font form here right now
+  webkit has a css2 implementation in 10.1.1. regarding to that spec */
```

Comment:
What is the actual issue? `webkit has a css2 implementation` is a confusing comment -- is it a bug in a particular part of css3 shorthand value parsing? Or something else?